diff --git a/node_modules/expo-updates/android/build.gradle b/node_modules/expo-updates/android/build.gradle
index 0994c5f..f5854f4 100644
--- a/node_modules/expo-updates/android/build.gradle
+++ b/node_modules/expo-updates/android/build.gradle
@@ -15,7 +15,7 @@ buildscript {
       if (kotlinVersion == "2.1.20") {
         return "2.1.20-2.0.1"
       } else if (kotlinVersion == "2.0.21") {
-        return "2.0.21-1.0.28"
+        return "2.1.20-1.0.32"
       } else if (kotlinVersion == "1.9.25") {
         return "1.9.25-1.0.20"
       }
