diff --git a/node_modules/expo-updates/android/build.gradle b/node_modules/expo-updates/android/build.gradle
index 0994c5f..3b0cdb1 100644
--- a/node_modules/expo-updates/android/build.gradle
+++ b/node_modules/expo-updates/android/build.gradle
@@ -13,7 +13,7 @@ buildscript {
       def kotlinVersion = rootProject["kotlinVersion"]
       
       if (kotlinVersion == "2.1.20") {
-        return "2.1.20-2.0.1"
+        return "2.1.20-1.0.32"
       } else if (kotlinVersion == "2.0.21") {
         return "2.0.21-1.0.28"
       } else if (kotlinVersion == "1.9.25") {
